server/game/cards/06-TLS/RedHorsesTail.js

```diff
@@ -0,0 +1,48 @@
+const GameActions = require('../../GameActions/index.js');
+const SpellCard = require('../../spellcard.js');
+
+class RedHorsesTail extends SpellCard {
+    setupCardAbilities(ability) {
+        this.spellAction({
+            title: 'Noon: Boot a Dude',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Choose a dude',
+                cardCondition: { 
+                    condition: card => card.gamelocation === this.parent.gamelocation
+                },
+                cardType: ['dude'],
+                gameAction: 'boot'
+            },
+            difficulty: context => context.target.value,
+            onSuccess: (context) => {
+                this.game.resolveGameAction(GameActions.bootCard({ card: context.target }), context);
+                this.game.addMessage('{0} uses {1} to boot {2}', context.player, this, context.target);
+            },
+            source: this
+        });
+
+        this.spellAction({
+            title: 'Shootout: Send Dude Home Booted',
+            playType: 'shootout',
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Choose an opposing dude',
+                cardCondition: { location: 'play area', controller: 'opponent', participating: true },
+                cardType: ['dude'],
+                gameAction: ['sendHome', 'boot']
```

Comment:
FYI: "boot" is redundant here as the check for boot and move is part of the "sendHome" check.